src/screens/explore/HostProfile/HostProfile.tsx

```diff
@@ -0,0 +1,114 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { StatusBar } from 'expo-status-bar';
+import { useCallback, useEffect } from 'react';
+import {
+  RefreshControl,
+  SafeAreaView,
+  ScrollView,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import { useSelector } from 'react-redux';
+import { Icon, Loader } from 'src/components';
+import { ScreenTemplate } from 'src/components/templates';
+import { RootStackParamList } from 'src/navigation';
+import { useAppDispatch } from 'src/store';
+import { getColors, getHostProfile, getHostProfileLoader } from 'src/store/selectors';
+import { AsyncThunks } from 'src/store/thunks';
+import { WHITE_100, LEVEL_1 } from 'src/styles';
+import { IconName } from 'src/types';
+
+import HostAbout from './components/HostAbout';
+import HostListings from './components/HostListings';
+import HostMainCard from './components/HostMainCard';
+import HostReviews from './components/HostReviews';
+import HostVerifiedInfo from './components/HostVerifiedInfo';
+
+type Props = NativeStackScreenProps<RootStackParamList, 'HostProfile'>;
+
+const HostProfile = ({ route, navigation }: Props) => {
+  const { hostId } = route.params;
+
+  const host = useSelector(getHostProfile);
+  const loading = useSelector(getHostProfileLoader);
+  const dispatch = useAppDispatch();
+  const colors = useSelector(getColors);
+
+  const fetchHostProfile = useCallback(async () => {
+    const response = await dispatch(AsyncThunks.getHostProfile(hostId));
+
+    if (!response.payload?.success) {
+      navigation.goBack();
+    }
+  }, [dispatch, hostId, navigation]);
+
+  useEffect(() => {
+    fetchHostProfile();
+  }, [fetchHostProfile]);
+
+  return (
+    <ScreenTemplate>
+      <SafeAreaView>
```

Comment:
why we need SafeAreaView and StatusBar here? if there is a problem with Statusbar, it is not the best option